### DISCUSSION TITLE
Semantic commit messages?

### DISCUSSION THREAD

**Discussion by christopher-cudennec** [dropwizard/dropwizard#5291] [Development]
Hey Dropwizard Team!

While trying out your latest 2.1 release I realized that it's really hard to tell which changes are important for us as consumers of your library. What about using semantic (or conventional) commit messages that contain a meaning, e.g. "feat", "fix", "core", ... 

It would be much easier to decide if a new release contains important changes.

Cheers, 

Christopher

**Comment by zUniQueX** [comment]
Hi @christopher-cudennec. I think that's a good idea overall, but I doubt it would be practical for Dropwizard. Many new features and fixes are contributed by the community and enforcing a convention for commit messages would cause overhead for both contributors and reviewers.
